apps/frontend/src/enums/Branches.ts

```diff
@@ -0,0 +1,5 @@
+export enum Branches {
```

Comment:
For something like this where there's no 'display' value or a reason to have two versions of each option, I'd prefer to just use a union type like 
`type Branches = 'CONFIRMATION' | 'FORM' | 'END';`
Not a necessary change though.

Reply:
Updated in most recent commit